tests/plugins/test_rpmqa.py

```diff
@@ -163,10 +173,10 @@ def test_rpmqa_plugin(caplog, docker_tasker, base_from_scratch, list_rpms_from_s
         assert workflow.image_components == parse_rpm_output(ignore_autogenerated["package_list"])
 
 
-@pytest.mark.parametrize('packages_exists', [True, False])
```

Comment:
unit test should test file existence for os.listdir

Reply:
updated the test, I extracted this part to separate testcase to be more straightforward and easier to follow.
You made a very good point as this caught a error when returning None from function gather_output but not from plugin itself.